### PR TITLE
fixing uint32_t error

### PR DESCRIPTION
**Pull request by vishweshjatala**
Fixed it locally. @chenxuhao and @ddn0 can please verify if it builds successfully.

### CODE REVIEW

File: lonestar/analytics/gpu/sgd/SGDGraphCu.h [hunk 1]
```diff
@@ -151,21 +151,21 @@ struct SGD_LC_LinearArray_Undirected_Graph {
     // node_data
     memset(node_data(), 0, sizeof(unsigned int) * _num_nodes);
     for (unsigned int i = 0; i < _num_edges; ++i) {
-      out_neighbors()[i] = *le64toh(outs + i);
```
Comment (ddn0) [inline lonestar/analytics/gpu/sgd/SGDGraphCu.h:154]: how did the previous version build at all?

- perhaps `le32toh(outs[i])` instead (and throughout).

Comment (vishweshjatala) [reply lonestar/analytics/gpu/sgd/SGDGraphCu.h:154]: Not sure. @chenxuhao  is owning that app and was building it. He can answer better.

Comment (ddn0) [reply lonestar/analytics/gpu/sgd/SGDGraphCu.h:154]: are we going to keep the `*(outs +i)` form or change it to `outs[i]`?

Comment (vishweshjatala) [reply lonestar/analytics/gpu/sgd/SGDGraphCu.h:154]: Yes, I am going to change it. Thanks

**Comment by vishweshjatala** [comment]
@insertinterestingnamehere all the builds except Sanitize pass. Can you please assign the issue for  failure of the Sanitize build to the appropriate person? This is not related to the GPU. Also, please merge the PR into git hub when you are fine.

**Pull request merged**